exchanges/poloniex/poloniex_types.go

```diff
@@ -66,6 +66,27 @@ type TradeHistory struct {
 	Total         float64 `json:"total,string"`
 }
 
+// OrderStatus holds order status data
+type OrderStatus struct {
+	Result  map[string]interface{} `json:"result"`
+	Success int64                  `json:"success"`
+}
+
+// OrderTrade holds order trade data
+type OrderTrade struct {
```

Comment:
https://docs.poloniex.com/#returntradehistory-private
startingAmount isnt a part of dat

Reply:
Poloniex request path: https://poloniex.com/tradingApi
Poloniex request header [Sign]: []
Poloniex request header [Content-Type]: [application/x-www-form-urlencoded]
Poloniex request header [Key]: []
Poloniex request type: POST
Poloniex request body: command=returnOrderTrades&nonce=1607558318602531900&orderNumber=18365912899
HTTP status: 200 OK, Code: 200
Poloniex raw response: [{"globalTradeID":326981405,"tradeID":660369,"currencyPair":"BTC_OMG","type":"sell","rate":"0.00173628","amount":"49.40355195","total":"0.08577839","fee":"0.00150000","date":"2018-01-14 23:34:50"}
**There doesn't seem to be a field startingAmount in the returned payload from this endpoint.**

Reply:
Yes, thanks!